desktop/README.md

```diff
@@ -3,3 +3,25 @@
 [![Docker Hub](https://img.shields.io/docker/pulls/qgis/qgis)](https://hub.docker.com/r/qgis/qgis)
 
 A simple QGIS desktop Docker image, pushed on [Docker Hub](https://hub.docker.com/r/qgis/qgis).
+
+## Python Plugin testing
+
+This image can be used to set up a testing environment for plugins, e.g.
+
+```
+PLUGIN_DIR=<plugin_name> # e.g. PLUGIN_NAME=valhalla
```

Comment:
I tested this with https://github.com/nilsnolde/valhalla-qgis-plugin, should work similarly for all plugins